config/main.py

```diff
@@ -163,12 +163,18 @@ def breakout_warnUser_extraTables(cm, final_delPorts, confirm=True):
     try:
         # check if any extra tables exist
         eTables = cm.tablesWithOutYang()
-        if len(eTables):
-            # find relavent tables in extra tables, i.e. one which can have deleted
-            # ports
-            tables = cm.configWithKeys(configIn=eTables, keys=final_delPorts)
-            click.secho("Below Config can not be verified, It may cause harm "\
-                "to the system\n {}".format(json.dumps(tables, indent=2)))
+        if eTables == None or len(eTables) == 0:
```

Comment:
Do we need check both? Or should "if not eTables:" can just do?  Same thing for below check for tables.

Reply:
these dict are always created, so len() check only will do. I will change the check.